parity/cli/mod.rs

```diff
@@ -1204,17 +986,6 @@ struct PrivateTransactions {
 #[serde(deny_unknown_fields)]
 struct Ui {
```

Comment:
can this be removed?

Reply:
I believe this is used by signer

Reply:
Can you elaborate? If it is, and Signer is going to be around still, maybe it's good to add a comment here to that effect?